test/webhookTest.js

```diff
@@ -0,0 +1,45 @@
+var assert = require('assert');
+var sinon = require('sinon');
+var webhook = require('../lib/webhook');
+
+describe('WebHook', function() {
+    var parser;
+    var fixer;
+    var commiter;
+    var next;
+
+    beforeEach(function() {
+        parser = sinon.spy();
+        fixer = sinon.spy();
+        commiter = sinon.spy();
+        next = {};
+    });
+
+    describe('parser', function() {
+        it('should be called every time', function *() {
+            yield webhook(parser, fixer, commiter)(next);
+            assert(parser.called);
+        });
+    });
+
+    describe('fixer', function() {
+        it('should be called every time', function *() {
+            yield webhook(parser, fixer, commiter)(next);
+            assert(fixer.called);
+        });
+    });
+
+    describe('commiter', function() {
+        it('should be called if fixer\'s content is not null', function *() {
+            fixer = sinon.stub().returns(42);
+            yield webhook(parser, fixer, commiter)(next);
```

Comment:
for the next PRs you can document yourself on the lib `co` and its method `wrap` to use a give a custom `this` to your webhook and therefore mock the request.